src/core/server/saved_objects/migrations/initial_state.test.ts

```diff
@@ -108,6 +113,10 @@ describe('createInitialState', () => {
       },
       versionAlias: '.kibana_task_manager_8.1.0',
       versionIndex: '.kibana_task_manager_8.1.0_001',
+      migrationDocLinks: {
+        resolveMigrationFailures:
+          'https://www.elastic.co/guide/en/kibana/test-branch/resolve-migrations-failures.html',
```

Comment:
This points to the full page. We can link to specific sections from the migrations error message as a follow-up.